libs/sdk/src/lib/services/posts.service.ts

```diff
@@ -17,14 +17,14 @@ import { ResourceService } from './resource.service';
 })
 export class PostsService extends ResourceService<any> {
   private defaultPostsFilters: GeoJsonFilter = {
-    order: 'desc',
-    orderby: 'created',
-    set: '',
-    order_unlocked_on_top: true,
-    reactToFilters: true,
-    'source[]': [],
+    // order: 'desc',
```

Comment:
Is this testing values?

Reply:
Yes. Filters is not ready yet, so in order to display posts, I need to set them statically.